### PR TITLE
Fix a bunch of docs syntax

### PR DESCRIPTION
**Pull request by jonashaag**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] Ensure all linting tests pass, see [here](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit) for how to run them
- [ ] whatsnew entry

From the Sphinx docs (https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionadded):

> Note that there must be no blank line between the directive head and the explanation; this is to make these blocks visually continuous in the markup.

### CODE REVIEW

**Review by phofl — COMMENTED** [review]
Could you show screen shots please?

Edit: Before and after

**Comment by jonashaag** [comment]
Actually it doesn't seem to make a difference for most of the docs so I reverted most of the changes.

File: pandas/io/excel/_base.py [hunk 1]
```diff
@@ -1217,14 +1219,15 @@ class ExcelFile:
            - Otherwise if ``path_or_buffer`` is in xlsb format,
              `pyxlsb <https://pypi.org/project/pyxlsb/>`_ will be used.
 
-           .. versionadded:: 1.3.0
+       .. versionchanged:: 1.3.0
```
Comment (phofl) [inline pandas/io/excel/_base.py:1222]: Please show screenshots, this looks to me as you changed the format here

Comment (jonashaag) [reply pandas/io/excel/_base.py:1222]: Will do, in the meantime have a look here, the layout is really messed up https://pandas.pydata.org/docs/reference/api/pandas.read_excel.html?highlight=read_excel#pandas.read_excel

File: doc/source/user_guide/10min.rst [hunk 2]
```diff
@@ -731,15 +731,15 @@ The :meth:`~plt.close` method is used to `close <https://matplotlib.org/3.1.1/ap
...
+   ts.plot()
 
 If running under Jupyter Notebook, the plot will appear on :meth:`~ts.plot`.  Otherwise use
 `matplotlib.pyplot.show <https://matplotlib.org/3.1.1/api/_as_gen/matplotlib.pyplot.show.html>`__ to show it or
 `matplotlib.pyplot.savefig <https://matplotlib.org/3.1.1/api/_as_gen/matplotlib.pyplot.savefig.html>`__ to write it to a file.
 
 .. ipython:: python
 
-   plt.show();
+   plt.show()
```
Comment (jreback) [inline doc/source/user_guide/10min.rst:742]: why do these need changing?

**Comment by jreback** [comment]
@jonashaag can you merge master and address comments

**Comment by jreback** [comment]
closing as stale (still work take if rebased)

**Pull request closed without merging**